src/telemetry/telemetry.bif

```diff
@@ -548,3 +549,19 @@ function Telemetry::__dbl_histogram_sum%(val: opaque of dbl_histogram_metric%):
 		return zeek::make_intrusive<DoubleVal>(0.0);
 		}
 	%}
+
+# Fix types?
```

Comment:
I don't fully recall, but I believe we indeed can't do this better than an `any_vec`.